### ISSUE TITLE
Is there a way to use a validation dataset?

### ISSUE THREAD

**Issue by swcrazyfan** [minimaxir/aitextgen#142]
I'm experimenting with fine-tuning, but I've recently realized loss can decrease, but the model overfits. Therefore, it can actually become not as good of a model. Do you have a way to include a validation dataset to check for overfitting?

Thanks!
